# HG changeset patch
# User "Yann E. MORIN" <yann.morin.1998@anciens.enib.fr>
# Date 1322159917 -3600
# Node ID 66ef6171f65ea7b70f49118dccbd687561cedd47
# Parent  e6b5bd48194274dc184c37091d6ff2b3c1ff305e
arch/arm: ARM supports setting the floating point type

Signed-off-by: "Yann E. MORIN" <yann.morin.1998@anciens.enib.fr>

diff --git a/config/arch/arm.in b/config/arch/arm.in
--- a/config/arch/arm.in
+++ b/config/arch/arm.in
@@ -9,6 +9,7 @@
 ## select ARCH_SUPPORTS_WITH_ARCH
 ## select ARCH_SUPPORTS_WITH_CPU
 ## select ARCH_SUPPORTS_WITH_TUNE
+## select ARCH_SUPPORTS_WITH_FLOAT
 ## select ARCH_SUPPORTS_WITH_FPU
 ## select ARCH_SUPPORTS_SOFTFP
 ##
